packages/dev/test/Flows.test.ts

```diff
@@ -18,27 +18,33 @@ import { GSNConfig } from '@opengsn/provider/dist/GSNConfigurator'
 import { registerForwarderForGsn } from '@opengsn/common/dist/EIP712/ForwarderUtil'
 import { defaultEnvironment } from '@opengsn/common/dist/Environments'
 import { ether } from '@opengsn/common'
-
-const TestRecipient = artifacts.require('TestRecipient')
-const TestPaymasterEverythingAccepted = artifacts.require('TestPaymasterEverythingAccepted')
-const TestPaymasterPreconfiguredApproval = artifacts.require('TestPaymasterPreconfiguredApproval')
-
-const StakeManager = artifacts.require('StakeManager')
-const Penalizer = artifacts.require('Penalizer')
-const Forwarder = artifacts.require('Forwarder')
+import Web3 from 'web3'
 
 const options = [
   {
     title: 'Direct-',
     relay: false
   },
   {
-    title: 'Relayed-',
-    relay: true
+    title: 'Legacy Relayed-',
+    relay: true,
+    type: 0
+  },
+  {
+    title: 'Type 2 Relayed-',
+    relay: true,
+    type: 2
   }
 ]
 
 options.forEach(params => {
+  const TestRecipient = artifacts.require('TestRecipient')
```

Comment:
any special reason to move those "require" into the "foreach" loop? 
we usually keep them at the top of the page.

Reply:
There was a reason before, now no